components/strava/strava.app.mjs

```diff
@@ -0,0 +1,128 @@
+import { axios } from "@pipedream/platform";
+export default {
+  type: "app",
+  app: "strava",
+  methods: {
+    _getHeaders(headers = {}) {
+      return {
+        "Authorization": `Bearer ${this.$auth.oauth_access_token}`,
+        "Content-Type": "application/json",
+        "user-agent": "@PipedreamHQ/pipedream v0.1",
+        ...headers,
+      };
+    },
+    _getUrl(path) {
+      return `https://www.strava.com/api/v3${path}`;
+    },
+    async _makeRequest({
+      $,
+      path,
+      headers,
+      ...otherConfig
+    } = {}) {
+      const config = {
+        url: this._getUrl(path),
+        headers: this._getHeaders(headers),
+        ...otherConfig,
+      };
+      return axios($ ?? this, config);
+    },
+    async getAuthenticatedAthlete({
+      $,
+      ...otherConfig
+    } = {}) {
+      return await this._makeRequest({
+        $,
+        method: "GET",
+        path: "/athlete",
+        params: otherConfig,
+      });
+    },
```

Comment:
You can refactor even more these functions like
```suggestion
    async getAuthenticatedAthlete(args = {}) {
      return this._makeRequest({
        method: "GET",
        path: "/athlete",
        ...args,
      });
    },
```
Then if you want to use this function from a component just do it like 
```javascript
await this.strava.getAuthenticatedAthlete({
  $,
  params: {...}
});
```

Reply:
done